solvebio/errors.py

```diff
@@ -53,7 +53,9 @@ def __init__(self, message=None, response=None):
                 else:
                     self.message += '\nError (%s): ' % k
 
-                if isinstance(v, list):
+                # Join lists of errors by commas, but if
```

Comment:
maybe we should just print v at this point

Reply:
ya....